lib/elixir/test/elixir/process_test.exs

```diff
@@ -68,6 +68,15 @@ defmodule ProcessTest do
     assert_receive :hello
   end
 
+  test "send_after/4 with absolute time sends message one expired" do
+    time =
+      System.monotonic_time
+      |> System.convert_time_unit(:native, :milliseconds)
```

Comment:
We can remove `convert_time_unit/3` call and use `System.monotonic_time(:milliseconds)`.